certified-connectors/Fliplet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,408 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Fliplet REST APIs",
+    "description": "This definition contains a list of action on Fliplet data sources",
```

Comment:
@squallstar all files should be free of grammatical and syntax errors. here you have "...list of action...", maybe you meant to say "...list of actions.."?